test/Basics/label1.js

```diff
@@ -4,7 +4,7 @@
 //-------------------------------------------------------------------------------------------------------
 
 function f() {
-    // Label in parenthesis is bad syntax, but we allow it. Verify consistency in deferred parsing.
+    // Label in parenthesis is bad syntax. Verify consistency in deferred parsing.

```

Comment:
If this syntax is now disallowed, shouldn't we have a parse error, or would this be something that now throws at runtime, so it's okay to have it in a defer-parsed function?